### ISSUE TITLE
tree fitting exercise doesn't show

### ISSUE THREAD

**Issue by andreashandel** [epid8060fall2019/meganlott-dataanalysis#1]
the link on the website doesn't work, it takes me to the previous exercise

**Comment by meganejlott** [comment]
Oops! Added in the wrong link in the navbar. Updated now! Thank you!

**Comment by meganejlott** [comment]
Oui. Just checked through the website again, and it was having the same issue. Finally updates with Tree Fitting Exercise in full! Thank you.
